README.md

```diff
@@ -22,22 +22,22 @@ This library is more suited to ETL than interactive queries, since large amounts
 
 ## Installation
 
-This library requires Apache Spark 1.4+ and Amazon Redshift 1.0.963+.
+This library requires Apache Spark 2.0+ and Amazon Redshift 1.0.963+.
 
 You may use this library in your applications with the following dependency information:
 
 **Scala 2.10**
 ```
 groupId: com.databricks
 artifactId: spark-redshift_2.10
-version: 0.6.0
+version: 0.7.0
```

Comment:
I'd change these versions to `2.0.0-SNAPSHOT`. Also, make sure to update `version.sbt` in the root of this repo.